lib/api/src/externals/function.rs

```diff
@@ -140,6 +143,12 @@ impl Function {
         let vmctx = VMFunctionEnvironment {
             host_env: Box::into_raw(Box::new(dynamic_ctx)) as *mut _,
         };
+        // TODO: look into removing transmute by changing API type signatures
```

Comment:
How we can change the API?

Reply:
There's no good way to do this that I can think of yet. The only way to avoid the transmutes would be to to make the `WasmerEnv::init_with_instance` function take `void*` directly... but that seems like a bad trade off.